core/app/models/spree/pricers/conservative_line_item_pricer.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Pricers
+    class ConservativeLineItemPricer
+      class VariantMissing < StandardError; end
+      class CurrencyMissing < StandardError; end
+
+      attr_reader :line_item
+
+      def initialize(line_item)
+        @line_item = line_item
+      end
+
+      def price
```

Comment:
I don't want to use a `Spree::Price`, as that would imply having the currency stored redundantly AGAIN. :)